### PR TITLE
Implemented account_muted and feature mutehistory command

### LINKED ISSUE #752: worldserver closes  (areatrigged db missing)
I just compiled a fresh azerothcore and I imported the database and the updates too. And the world server at the end says: ERROR: Your database structure is not up to date. Please make sure you've executed all queries in the sql/updates folders. 
ERROR: Table "acore_world.areatrigger" doesn't exist

### PR DESCRIPTION
**Pull request by Nefertumm**
**Changes proposed:**

-  Implemented the table and the command. Imported from TrinityCore/TrinityCore@15a9c6729ee274044b2ba78f8b9ad3c167c94bd1

-  Fix the sql_rev, causing problems in the updates.

**Target branch(es):** master.

**Issues addressed:** Updates #236. Closes #752, #757.
**Tests performed:** Tested and working as intended.

**Known issues and TODO list:** -

### CODE REVIEW

File: data/sql/updates/db_world/2017_12_08_00.sql [hunk 1]
```diff
@@ -5,7 +5,7 @@ CREATE PROCEDURE updateDb ()
 proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';
 SELECT COUNT(*) INTO @COLEXISTS
 FROM information_schema.COLUMNS
-WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2017_12_08_00';
+WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2017_11_21_00';
```
Comment (Yehonal) [inline data/sql/updates/db_world/2017_12_08_00.sql]: why these changes?

Comment (Nefertumm) [reply data/sql/updates/db_world/2017_12_08_00.sql]: Because this sql was still causing problem with the updates since b527073fd84f335fa20ac3943618448f9a43e414. You forget to change that column_name :P

**Comment by ShinDarth** [comment]
@Nefertumm thanks for the PR and @jllllll thanks for testing it
